openapi.yaml

```diff
@@ -276,11 +270,16 @@ paths:
         '500':
           description: Internal Server Error
       x-tira: true
-    parameters:
-      - $ref: '#/components/parameters/userId'
     post:
       summary: Add mood to diary
       operationId: post-diary-userId
+      parameters:
+        - in: header
+          name: X-userId
```

Comment:
I think this should not be exposed in the spec as this is an internal header only that clients won't have to set.